### PR TITLE
Wrapper classes removed. Fixes #31

### LINKED ISSUE #31: Remove wrapper class for Customer
`Fortnox::API::Customer` should be removed. Instead, use `Fortnox::API::Model::Customer`, `Fortnox::API::Validator::Customer` and `Fortnox::API::Repository::Customer`.

### PR DESCRIPTION
**Pull request by ehannes**
Fixes #31 

With the new architecture for the gem, wrapper classes are not
needed anymore.

To load all dependencies, api requires three new classes:
Fortnox::API::Models, Fortnox::API::Repositories,
Fortnox::API::Validators. These classes then loads required model,
repository and validator classes.

### CODE REVIEW

File: lib/fortnox/api/models.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require "fortnox/api/models/customer"
+
+module Fortnox
```
Comment (d-Pixie) [inline lib/fortnox/api/models.rb]: This is not needed.

File: lib/fortnox/api/repositories.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require "fortnox/api/repositories/customer"
+
+module Fortnox
```
Comment (d-Pixie) [inline lib/fortnox/api/repositories.rb]: Same here.

File: lib/fortnox/api/validators.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require "fortnox/api/validators/customer"
+
+module Fortnox
```
Comment (d-Pixie) [inline lib/fortnox/api/validators.rb]: And here ... :)

**Comment by d-Pixie** [comment]
If we had used the `class Fortnox::API::Model::Customer ... end` syntax to declare the models/validators/repositories we would have had to initialize the parent modules in the path first. But since we use explicit module definitions in each of the classes:
```ruby
module Fortnox
  module API
    module Model
      class Customer
        ...
      end
    end
  end
end
```
Each class declaration essentially redeclares it's module chain this way, so you never have to worry about when the initilisation is run...

**Comment by ehannes** [comment]
I'm not sure I'm following...

If `lib/fortnox/api/models.rb` does not need this code:
```require "fortnox/api/models/customer"
+
+module Fortnox
```

**Comment by d-Pixie** [comment]
No, the commented line is the first line in this block:
```ruby
module Fortnox
  module API
    module Model
    end
  end
end
```
That's what's not needed, since the required file already declares this anyway.

**Comment by ehannes** [comment]
Ok. So should I remove the whole block? And just keep the first line in `lib/fortnox/api/models.rb`, like this:

```
require "fortnox/api/models/customer"
```

**Comment by d-Pixie** [comment]
Yep :)

**Pull request merged**